docs/.wordlist.txt

```diff
@@ -47,3 +47,22 @@ YAML
 STSStack
 sshuttle
 vpn
+behavior
```

Comment:
`behavior` is US spelling; at Canonical we'd use the UK spelling of `behaviour`. ;)